gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineErrorProne.java

```diff
@@ -209,11 +210,11 @@ private static void configureErrorProneOptions(
             JavaCompile javaCompile,
             ErrorProneOptions errorProneOptions) {
 
-        if (project.hasProperty(DISABLE_PROPERTY) || IntellijSupport.isRunningInIntellij()) {
-            log.info("Disabling baseline-error-prone for {} due to {}", project, DISABLE_PROPERTY);
-            errorProneOptions.getEnabled().set(false);
-        } else {
+        if (project.hasProperty(ENABLE_PROPERTY)) {
             errorProneOptions.getEnabled().set(true);
```

Comment:
We can just rely on the plugin to have an appropriate default value here:
https://github.com/tbroyer/gradle-errorprone-plugin/blob/8d1f44dcef0ee68b5712ab1d319acfd2826db2a6/src/main/kotlin/net/ltgt/gradle/errorprone/ErrorProneOptions.kt#L24-L25